static/css/page-plain.less

```diff
@@ -12,187 +12,185 @@
-    margin-bottom: 45px;
-    margin-top: 20px;
+  p {
+    font-size: .75em;
+    padding: 0;
+    margin: 0;
   }
-  h1 {
-    font-size: 1.5em;
-    margin: 0 0 15px;
+}
+ul.link {
+  margin-bottom: 50px;
+  li {
+    font-size: 1.125em;
   }
-  div.navBorrow {
-    margin: 50px 0 10px;
+}
+/* stylelint-disable selector-max-specificity */
+#faq {
+  h4 {
     font-size: .75em;
+    color: @black;
+    text-transform: uppercase;
+    margin-bottom: 0;
   }
-  .panel {
-    background-color: @white;
-    border: 1px solid @beige-two;
-    padding: 30px 20px;
-    margin-bottom: 20px;
-  }
-  .formElement {
-    float: none;
+  p {
+    font-size: .75em;
+    color: @dark-grey;
   }
-  .panel h3 {
-    font-size: 1.875em;
+}
+/* stylelint-enable selector-max-specificity */
+p.help {
+  font-size: 14px;
+}
+div.alert {
+  background: @light-yellow url(/images/icons/icon_alert-gold.png) no-repeat 15px 50% !important;
+  margin-bottom: 10px;
+  margin-top: 20px;
+}
+div.stop {
+  background: @light-yellow url(/images/icons/icon_stop.png) no-repeat 15px 50% !important;
+  margin-bottom: 45px;
+  margin-top: 20px;
+}
+div.info {
+  background: @light-yellow url(/images/icons/icon_info.png) no-repeat 17px 50% !important;
+  margin-bottom: 45px;
+  margin-top: 20px;
+}
+h1 {
+  font-size: 1.5em;
+  margin: 0 0 15px;
+}
+div.navBorrow {
+  margin: 50px 0 10px;
+  font-size: .75em;
+}
+.panel {
+  background-color: @white;
+  border: 1px solid @beige-two;
+  padding: 30px 20px;
+  margin-bottom: 20px;
+}
+.formElement {
+  float: none;
+}
+.panel h3 {
+  font-size: 1.875em;
+}
+/* stylelint-disable selector-max-specificity */
+#borrowThis {
+  margin-bottom: 20px;
+}
+/* stylelint-enable selector-max-specificity */
+.borrow {
+  span.book {
+    font-size: 1em;
+    font-weight: 700;
+  }
+  span.author {
+    font-size: .875em;
+    color: @dark-grey;
+  }
+  span.publisher,
+  span.contributor {
+    color: @dark-grey;
+    font-size: .875em;
+  }
+  div.date {
+    font-size: .75em;
+    color: @grey;
   }
-  /* stylelint-disable selector-max-specificity */
-  #borrowThis {
-    margin-bottom: 20px;
+  span.expires {
+    font-size: 1em;
   }
-  /* stylelint-enable selector-max-specificity */
-  .borrow {
-    span.book {
-      font-size: 1em;
-      font-weight: 700;
-    }
-    span.author {
-      font-size: .875em;
-      color: @dark-grey;
-    }
-    span.publisher,
-    span.contributor {
-      color: @dark-grey;
-      font-size: .875em;
-    }
-    div.date {
-      font-size: .75em;
-      color: @grey;
-    }
-    span.expires {
-      font-size: 1em;
-    }
-    table {
-      border-bottom: none;
-    }
+  table {
+    border-bottom: none;
   }
-  // Specific to borrowTable component
-  /* stylelint-disable selector-max-specificity */
-  #borrowTable {
-    margin: 60px 0 25px;
+}
+// Specific to borrowTable component
+/* stylelint-disable selector-max-specificity */
+#borrowTable {
```

Comment:
Perhaps in a follow up we could move this into legacy-borrowTable-plain.less for consistency with `legacy-borrowTable-adminUser.less`.

The eventual goal would be to unify these styles with those in legacy-borrowTable-adminUser.less  and static/css/components/borrowTable.less